spdx-utils/src/main/kotlin/SpdxExpression.kt

```diff
@@ -54,6 +54,11 @@ sealed class SpdxExpression {
     }
 
     companion object {
+        /**
+         * The "WITH" keyword, used to concatenate a license with an exception.
+         */
+        const val WITH = "WITH"
```

Comment:
Shouldn't this rather be moved to `SpdxLicenseWithExceptionExpression`?

Reply:
Ping?

Reply:
I have put it here because then the keyword can be accessed from other code with `SpdxExpression.WITH` without having to know the subtype.

Reply:
But isn't it so that if you need `WITH`, you always do so in the context of an `SpdxLicenseWithExceptionExpression`? I.e. you already know the subtype? But I'm fine to keep it here.